src/include/cell.h

```diff
@@ -154,12 +154,17 @@ struct __wt_cell_unpack {
 
 	uint64_t v;			/* RLE count or recno */
 
-	uint64_t start_txn, stop_txn;	/* Value validity window */
-	wt_timestamp_t start_ts, stop_ts;
+	wt_timestamp_t	start_ts;	/* Value validity window */
+	uint64_t	start_txn;
+	uint64_t	stop_txn;
+	wt_timestamp_t	stop_ts;
```

Comment:
I am not able to understand why the `stop_txn` and `stop_ts` are interchanged here.

Reply:
Thanks, @bvpvamsikrishna. Fixed, pushed in 7d9942c.